src/os/bluestore/bluestore_tool.cc

```diff
@@ -810,6 +821,53 @@ int main(int argc, char **argv)
     jf.close_section();
     jf.flush(cout);
   }
+  else if (action == "create-bdev-label") {
+
+    std::vector<std::string> metadata_files = {
+        "bfm_blocks", "bfm_blocks_per_key", "bfm_bytes_per_block", "bfm_size",
+        "bluefs", "ceph_fsid", "ceph_version_when_created", "created_at",
+        "elastic_shared_blobs", "fsid", "keyring", "kv_backend", "magic",
+        "mkfs_done", "osd_key", "ready", "require_osd_release", "type", "whoami"
+    };
```

Comment:
List of metas is a good idea.
It is almost certain that we will add more metas in the future.
I think we should have some warning when there is a file (potentially meta) in the dir that we are not putting to bdev label. 
However, ignoring those extra files is OK overall.
If they are needed in meta one can always use CBT to set additional meta.